_locales/pl/messages.json

```diff
@@ -135,15 +135,27 @@
     "description": "Tab title for the image properties pop-up. Refers to data embedded within an image, such as Exif data."
   },
   "prop_header_width": {
-    "message": "80px",
+    "message": "80 pikseli",
```

Comment:
This actually doesn't need to be translated. It's so you can increase the size of the header column if you need to:

![image](https://cloud.githubusercontent.com/assets/760836/18235869/a506181a-72e5-11e6-8515-e5851c00495d.png)

It gets placed directly into the CSS, so it should stay "px".